### ISSUE TITLE
Error launching from DSi XL

### ISSUE THREAD

**Issue by federicochiesa** [Gericom/GBARunner2#59]
I can't get GBARunner2 to work on my DSi XL. I'm using Twilightmenu++ and I have put the bios in the right dircetory, tried multiple bioses, but all I get is a white screen with "BINF3DS sd card áß áß áß áß áß áß áß áß " written.

Am I doing something wrong?

**Comment by RocketRobz** [comment]
Update TWiLight Menu++ to the latest version, which is v8.5.1.

**Comment by federicochiesa** [comment]
Thanks for the advice, I just did that, and now the screen has changed, but it still doesn't start.
Now I have a black square in the top left corner of the screen and a horizontal grey bar across the upper part of the screen.
If you need I can send a picture.

**Comment by Gericom** [comment]
Hmm, I've seen that before, but it's not entirely clear to me when it happens tbh

**Comment by federicochiesa** [comment]
Is it possibly a bad GBA BIOS? I tried everything at this point, I might just be unlucky....

**Comment by Gericom** [comment]
No, the bios is not used yet at that point. Did you try reformatting

**Comment by federicochiesa** [comment]
Sorry, I'm a complete noob, do you mean just reformatting the SD card or reinstall unlaunch from scratch on my DSi?

**Comment by zacchi4k** [comment]
I occasionally get this issue too, and every time it can be solved by moving the GBA BIOS to another folder and moving it back

**Comment by Gericom** [comment]
Hmm, that's strange.

**Comment by zacchi4k** [comment]
I'm guessing it's some issue with how Gbarunner2 handles the FAT? It always happens after I place/modify files on the SD card (for example when updating TWiLightMenu++). The SD is known to be good (I used h2testw on it) and it is properly formatted

**Comment by Gericom** [comment]
It's weird, because I'm using a proper fat library now. That library is know to be working well afaik.

**Comment by zacchi4k** [comment]
I'm using the ARM7 build included with TWiLightMenu++. Prehaps I could try building from source and seeing if the issue persists with new versions

**Comment by Gericom** [comment]
Automatic builds are available from the github release page. But I doubt it would change anything as nothing related to fat handling has changed recently.

**Comment by hakiyomi** [comment]
i get this issue too, i stuck in white screen when tried to run gbarunner2

**Comment by federicochiesa** [comment]
With the latest Twilightmenu update I can now see [the main screen](https://imgur.com/a/UcK8dJA), but nothing works, it freezes right there.

**Comment by Gericom** [comment]
@federicochiesa That issue you metioned last was most likely #78 which is fixed

**Comment by RJMcDanielsJr** [comment]
I am in the same boat as feder.  I have tried three different DS's with multiple bios files.  I have put the file in both the GBA folder and outside the GBA folder on root.  I also downloaded the newest build twighlit menu and had no luck.  I am beyond stumped.  I wished someone could just send me the bios file that they use.

**Comment by Gericom** [comment]
Doesn't seem like an issue with the bios file to me

**Comment by zenseii** [comment]
Support for the open-source bios has been implemented by Gericom so problems with bioses should be solved, even if compatibility is better by supplying your own bios.
